app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -271,4 +291,33 @@ def availability_zones
       :ems_ref => persister.cloud_manager.uid_ems
     )
   end
+
+  def software_licenses_description(software_licenses)
+    ldesc = ""
+
+    if software_licenses
+      if software_licenses.ibmi_css
+        ldesc = "IBMi Cloud Storage Solution (ibmiCSS), "
+      end
+
+      if software_licenses.ibmi_pha
+        ldesc << "IBMi Power High Availability (ibmiPHA), "
+      end
+
+      if software_licenses.ibmi_rds
+        ldesc << if software_licenses.ibmi_rds_users
+                   "IBMi Rational Dev Studio (ibmiRDS) - (%d User Licenses)" % [software_licenses.ibmi_rds_users]
```

Comment:
This needs a trailing `", "`
```suggestion
                   "IBMi Rational Dev Studio (ibmiRDS) - (%d User Licenses), " % [software_licenses.ibmi_rds_users]
```

or you can just refactor as I mentioned in https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/246/files#r674185383